### ISSUE TITLE
Compilation errors with hyperref

### ISSUE THREAD

**Issue by f0k** [ismir/paper_templates#13]
Whenever I add
```latex
\usepackage{hyperref}
```
to the preamble of an ISMIR paper and run `pdflatex`, I get compilation errors like these:
```
! Extra \fi.
\Hy@writebookmark ... {#1}{#2}{#3}{#4}{#5}\fi \fi 
                                                  
l.133 \section{Introduction}
                            \label{sec:introduction}
```
In the following prompt, if I enter "R" to go into nonstopmode (or likewise, if I run `pdflatex -interaction nonstopmode`), compilation finishes fine and links to figures, sections, footnotes and URLs in the output file work, but citations are not linked to the respective bibliography entries (they do not turn into hyperrefs).

This has happened to me for every ISMIR paper. Until disproven, I blame `ismir.sty`, but I don't know how to start debugging this.

**Comment by f0k** [comment]
> but citations are not linked to the respective bibliography entries

Partial success: The version of `cite.sty` included in the template is too old. Replacing it with the current one (http://mirrors.ctan.org/macros/latex/contrib/cite/cite.sty) enables hyperlinked references.

The `Extra \fi` error remains.

**Comment by fzalkow** [comment]
As mentioned in [this issue](https://github.com/ismir/ismir2016-lbd/issues/35) you have disable bookmarks when including hyperref:

```
\usepackage[bookmarks=false]{hyperref}
```

**Comment by f0k** [comment]
For next ISMIR, we might want to include some commented lines in the preamble such as:
```latex
% Optional: To use hyperref, uncomment the following.
% \usepackage[bookmarks=false,hidelinks]{hyperref}
% Mind the bookmarks=false option; bookmarks are incompatible with ismir.sty.
```
Could we add this issue to a milestone `ismir2019` so it's not forgotten?

**Comment by stefan-balke** [comment]
resolved in #17
